### DISCUSSION TITLE
Stripe Checkout

### DISCUSSION THREAD

**Discussion by DragomirRhv** [cypress-io/cypress#9579] [Questions and Help]
I have to call stripe.redirectToCheckout to take the customer to stripe checkout page. But the frame is lost when stripe navigates to the page. I got the following code: 

`cy.intercept(
        'GET',
        'https://checkout.stripe.com/pay',

        {
            statusCode: 200,
            body: {
                message: 'Request successful',
                result: ['my-data']
            },
        }
    ).as('getSession')
cy.get('#checkout-btn').click()`

I have attached a screenshot of the result:
![Screenshot at 2020-12-09 01-14-23](https://user-images.githubusercontent.com/52378777/101553026-9a485a80-39bc-11eb-8ee3-4713658e5fa8.png)

Visually it is working but the frame is lost. I cannot do anything after that. Is there anyway just to intercept the request and to verify somehow, that it's going to redirect me to stripe domain page!?
If I use the code below:

`cy.intercept(
        'GET',
        'https://checkout.stripe.com/pay',
        (request) => {
             return request.redirect({location: 'http:0.0.0.0:3000/something'})
         }
    )`

It is loading for a while because I have a loader before the redirect and then the frame is lost again.

**Comment by DragomirRhv** [comment]
@jennifer-shehane

**Comment by jennifer-shehane** [comment]
What version of Cypress are you using? There's an issue with Stripe Checkout detailed here that may be affecting you https://github.com/cypress-io/cypress/issues/9447

**Reply by DragomirRhv** [reply]
The version I am using is the latest one 6.0.1 , I thought that this issue is fixed here. So as I understand I must use 5.5.0?
